### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by elliottsj**
See commit messages

### CODE REVIEW

File: eslintrc.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "brace-style": [2, "1tbs", { "allowSingleLine": true }],
     "camelcase": 2,
     "default-case": 2,
-    "dot-notation": [2, { "allowPattern": "^[a-z]+(_[a-z]+)+$" }],
+    "dot-notation": [2, { "allowPattern": "^[a-z]+(_+[a-z]+)+$"} ],
```
Comment (RickEyre) [inline eslintrc.json]: Why one or more underscore? What's the use case for that?

Comment (elliottsj) [inline eslintrc.json]: Some of our code is like `obj['date__close_date']`

Comment (RickEyre) [inline eslintrc.json]: Right.

**Comment by RickEyre** [comment]
:whale:

**Pull request merged**